retrieval/discovery/consul.go

```diff
@@ -264,9 +268,11 @@ func (cd *ConsulDiscovery) watchService(srv *consulService, ch chan<- *config.Ta
 			tags := cd.tagSeparator + strings.Join(node.ServiceTags, cd.tagSeparator) + cd.tagSeparator
 
 			srv.tgroup.Targets = append(srv.tgroup.Targets, clientmodel.LabelSet{
-				clientmodel.AddressLabel: clientmodel.LabelValue(addr),
-				ConsulNodeLabel:          clientmodel.LabelValue(node.Node),
-				ConsulTagsLabel:          clientmodel.LabelValue(tags),
+				clientmodel.AddressLabel:  clientmodel.LabelValue(addr),
+				ConsulNodeLabel:           clientmodel.LabelValue(node.Node),
+				ConsulTagsLabel:           clientmodel.LabelValue(tags),
+				ConsulServiceAddressLabel: clientmodel.LabelValue(node.ServiceAddress),
+				ConsulServicePortLabel:    clientmodel.LabelValue(node.ServicePort),
```

Comment:
Should we also add `node.Address` here? Technically a user could already extract it from `__address__` (which is set above), but the same is true for the `ServicePort`.